packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts

```diff
@@ -258,7 +258,7 @@ async function askAdditionalQuestions(context, authConfig, defaultAuthType, mode
   const advancedSettingsQuestion = {
     type: 'list',
     name: 'advancedSettings',
-    message: 'Do you want to configure advanced settings for the GraphQL API',
+    message: 'Do you want to configure advanced settings for the GraphQL API? (including enabling Amplify DataStore)',
```

Comment:
call it only "DataStore", we should be able to omit the Amplify here.

Reply:
i will defer to you on this, but i included "Amplify" intentionally because it makes it easier to google. if you prefer it shorter still, then feel free to accept this edit

```suggestion
    message: 'Do you want to configure advanced settings for the GraphQL API? (including enabling DataStore)',
```